pixels/endpoints/general.py

```diff
@@ -0,0 +1,178 @@
+import logging
+
+from fastapi import APIRouter, Depends, HTTPException, Request, Response
+
+from pixels.constants import Ratelimits, Sizes
+from pixels.models import GetSize, Message, Pixel
+from pixels.utils import auth, ratelimits
+
+log = logging.getLogger(__name__)
+router = APIRouter(tags=["Canvas Endpoints"], dependencies=[Depends(auth.JWTBearer())])
+
+
+@router.get("/size", response_model=GetSize)
+async def size(request: Request) -> GetSize:
+    """
+    Get the size of the Pixels canvas.
+
+    You can use the data this endpoint returns to build some cool scripts
+    that can start the ducky uprising on the canvas!
+
+    This endpoint doesn't require any authentication so don't worry
+    about the headers usually required.
```

Comment:
This is actually incorrect now? The dependency on JWTBearer() is router-wide, including this endpoint.

Reply:
Yea, true. may need to refactor this out, since we use get_size as an aliveness check, and we don't use auth.